modules/sso/class.jetpack-sso-helpers.php

```diff
@@ -22,6 +24,12 @@ static function should_hide_login_form() {
 		return (bool) apply_filters( 'jetpack_remove_login_form', get_option( 'jetpack_sso_remove_login_form', false ) );
 	}
 
+	/**
+	 * Returns a boolean value for whether logging in by matching the WordPress.com user email to a
+	 * Jetpack site user's email is allowed.
```

Comment:
Good point. I'm not to sure, especially since all but 1 of the methods was copied over from the SSO module. In that case, should the `@since` be `4.1.0`, or would it instead be when the method was originally created in the SSO module?